frontend/src/components/intake/ShasIntakeSchema.ts

```diff
@@ -0,0 +1,144 @@
+import { array, boolean, mixed, number, object, string } from 'yup';
+
+import {
+  ContactPreferenceList,
+  NumResidentialUnits,
+  ProjectRelationshipList,
+  YesNo,
+  YesNoUnsure
+} from '@/utils/constants';
+import { BASIC_RESPONSES } from '@/utils/enums';
+
+// Form validation schema
+const YesNoUnsureSchema = string().required().oneOf(YesNoUnsure);
+const prevYesSelected = (prevQuestion: string) => prevQuestion === BASIC_RESPONSES.YES;
```

Comment:
I'm not feeling the necessity of pulling this little bit out into it's own function?